src/test/java/io/fusionauth/samlv2/service/DefaultSAMLv2ServiceTest.java

```diff
@@ -111,6 +112,42 @@
 @SuppressWarnings({"unchecked"})
 @Test(groups = "unit")
 public class DefaultSAMLv2ServiceTest {
+  @Test
+  public void assertionDecryptionDefaults() throws Exception {
+    // If RSA-OAEP Digest and Mask Generation Function are not specified by XML, decryption should fall back to the defaults
+
+    // Build a known key pair.
+    KeyFactory factory = KeyFactory.getInstance("RSA");
+    // The public key is not required for this test, but it may be useful to modify or expand this test in the future
+    // PublicKey publicKey = factory.generatePublic(new RSAPublicKeySpec(new BigInteger("21734648244307152755738902242704624429675455693104061482953980655823499524284217582577935962219675181839097134429878676848067944269649003417313253763145613039845156858929146350893510281417425701635390227843218753386852942958087790126591910892081707753005524949329857277363222746280909051526362184081185954039703446436022345307092346517413518280909483768946131477611274390374625720745000173012484689181319542884541163003470909355448313533318136237678943263133529991715284549440616270148923866161198748312992261382455526114770464413102345807150728423473869759031086596301998397561122681012070445972165920288084712186321"), new BigInteger("65537")));
+    PrivateKey privateKey = factory.generatePrivate(
+        new RSAPrivateKeySpec(
+            new BigInteger("21734648244307152755738902242704624429675455693104061482953980655823499524284217582577935962219675181839097134429878676848067944269649003417313253763145613039845156858929146350893510281417425701635390227843218753386852942958087790126591910892081707753005524949329857277363222746280909051526362184081185954039703446436022345307092346517413518280909483768946131477611274390374625720745000173012484689181319542884541163003470909355448313533318136237678943263133529991715284549440616270148923866161198748312992261382455526114770464413102345807150728423473869759031086596301998397561122681012070445972165920288084712186321"),
```

Comment:
line not long enough. please add spacers ever 3 digits so we can count easier i.e. 1_000_000

(i kid, i kid)